### ISSUE TITLE
Adding related projects/licenses? 

### ISSUE THREAD

**Issue by jywarren** [ContributorCovenant/hippocratic-license#36]
https://cosli.eu/ has some similarities and I was curious if you're interested in PRs that add related project links? I'm happy to open one if so!

**Comment by CoralineAda** [comment]
Yes, that would be great! Also consider adding it here (duplication is intentional): https://github.com/contributorCovenant/ethicalsource/
